sdk/communication/Azure.Communication.Identity/CHANGELOG.md

```diff
@@ -10,6 +10,19 @@
 
 ### Other Changes
 
+## 1.4.0-beta.1 (2025-03-02-preview)
```

Comment:
This needs to be the actual date when we're planning to release it and it's used by the unified build+release pipeline. It's not the REST API version